cmake/compile_definitions/linux.cmake

```diff
@@ -218,7 +218,6 @@ list(APPEND PLATFORM_LIBRARIES
         Boost::dynamic_linking
         dl
         evdev
-        numa
```

Comment:
I forget the reason, but we had to add this when we switched to our custom ffmpeg build.

I will run the workflow to test if anything has changed.

Reply:
I haven't removed it entirely, just moved it to the `FFMPEG_PLATFORM_LIBRARIES`. See the second commit.